### PR TITLE
Startup minh

### PR DESCRIPTION
**Pull request by MikeHoang**

### CODE REVIEW

File: app/assets/javascripts/notification.js.erb [hunk 1]
```diff
@@ -13,6 +13,7 @@ channel.bind('commented', function(data) {
 });
 
 channel.bind('liked', function(data) {
+  console.log(data);
```
Comment (minhquan4080) [inline app/assets/javascripts/notification.js.erb]: remove console.log

File: app/views/user/ideas/show.html.erb [hunk 2]
```diff
@@ -223,6 +223,30 @@
 
 <script>
 $(document).ready(function() {
+  var idea_id = <%= @idea.id %>
+  channel.bind('like_per_views', function(data) {
+  if (data.idea == idea_id){
+   $.get('/api/idea/like/'+ data.idea, function(data){
+      console.log(data);
```
Comment (minhquan4080) [inline app/views/user/ideas/show.html.erb]: remove console.log

Comment (minhquan4080) [inline app/views/user/ideas/show.html.erb]: remove console.log

File: app/views/user/ideas/show.html.erb [hunk 3]
```diff
@@ -223,6 +223,30 @@
...
+   //$('.like-page').html(html);
+  }
+
+});
+
+  channel.bind('rating', function(data) {
+    console.log(data);
+  if (data.idea == idea_id){
+   $.get('/api/idea/rating/'+ data.idea, function(data){
+      console.log(data);
```
Comment (minhquan4080) [inline app/views/user/ideas/show.html.erb]: remove console.log

**Comment by minhquan4080** [comment]
@MikeHoang  please check my comment

File: app/assets/stylesheets/idea/style.css.scss [hunk 4]
```diff
@@ -9856,4 +9856,20 @@ textarea.form-control1.control2 {
...
+  width:14.333%;
+}
+.column-1 {
+    margin-left: -150px;
+}
+.column-2 {
+    margin-left: -25px;
+}
+.col-md-12.activity-rows1{
+  padding: 15px 0
```
Comment (minhquan4080) [inline app/assets/stylesheets/idea/style.css.scss]: missing ;

File: app/models/rate.rb [hunk 5]
```diff
@@ -4,4 +4,8 @@ class Rate < ActiveRecord::Base
 
   #attr_accessible :rate, :dimension
 
+  def self.get_rating_statistic(idea_id, star)
+
```
Comment (minhquan4080) [inline app/models/rate.rb]: remove empty line

File: app/views/user/ideas/show.html.erb [hunk 6]
```diff
@@ -223,6 +239,28 @@
...
+   });
+   //$('.like-page').html(html);
+  }
+
+});
+
+  channel.bind('rating', function(data) {
+  if (data.idea == idea_id){
+   $.get('/api/idea/rating/'+ data.idea, function(data){
+      console.log(data);
```
Comment (minhquan4080) [inline app/views/user/ideas/show.html.erb]: remove console.log

**Comment by minhquan4080** [comment]
@MikeHoang  please check my cmt. Thanks

**Comment by lhhieu89** [comment]
LGTM thanks

**Pull request merged**